app/src/main/java/org/fossasia/pslab/fragment/LAChannelModeFragment.java

```diff
@@ -197,14 +205,61 @@ public void onClick(View v) {
         return selectView;
     }
 
-    public interface OnChannelSelectedListener {
-        void channelSelectedNowAnalyze(Bundle params);
+    @SuppressLint("ResourceType")
+    public void howToConnectDialog(String title, String intro, int iconID, String desc) {
+        try {
+            final android.support.v7.app.AlertDialog.Builder builder = new android.support.v7.app.AlertDialog.Builder(getContext());
+            LayoutInflater inflater = getLayoutInflater();
+            View dialogView = inflater.inflate(R.layout.custom_dialog_box, null);
+
+            final SharedPreferences settings = getActivity().getSharedPreferences(PREFS_NAME, 0);
+            String skipMessage = settings.getString("skipMessage", "NOT checked");
+            dontShowAgain = (CheckBox) dialogView.findViewById(R.id.toggle_show_again);
+            final TextView tv1 = (TextView) dialogView.findViewById(R.id.custom_dialog_text);
+            final TextView tv2 = (TextView) dialogView.findViewById(R.id.description_text);
+            final ImageView iv = (ImageView) dialogView.findViewById(R.id.custom_dialog_schematic);
```

Comment:
Good Java practice is to use meaningful variable names instead of these type of naming